### PR TITLE
[ci] [docker] Update components in Docker image

### PR DESCRIPTION
**Pull request by ejgallego**
We bump ounit, odoc, and lablgtk3 ; so far this is routine maintenance.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ stages:
 variables:
   # Format: $IMAGE-V$DATE [Cache is not used as of today but kept here
   # for reference]
-  CACHEKEY: "bionic_coq-V2019-03-01-V43"
+  CACHEKEY: "bionic_coq-V2019-03-14-V14"
```
Comment (SkySkimmer) [inline .gitlab-ci.yml]: It took 14 tries to get something that works?

**Review by SkySkimmer — APPROVED** [review]

**Review by SkySkimmer — CHANGES_REQUESTED** [review]
You should keep the example overlay (`10185-SkySkimmer-instance-no-bang.sh`)

File: .gitlab-ci.yml (hunk 1 above)
Comment (ejgallego) [reply .gitlab-ci.yml]: For a while I've been just picking a random number as to avoid possible clashes with other PRs.

**Comment by ejgallego** [comment]
Ups, overlay kept.

**Comment by ejgallego** [comment]
Maybe we should just have some "00000-dont-delete.sh" overlay, the example is not needed as we have the script.

**Review by SkySkimmer — APPROVED** [review]

File: .gitlab-ci.yml (hunk 1 above)
Comment (SkySkimmer) [reply .gitlab-ci.yml]: We could use date-PR number-version

**Pull request merged**